### DISCUSSION TITLE
Curl not returning indices

### DISCUSSION THREAD

**Discussion by juddcbrown** [Security-Onion-Solutions/securityonion#4992] [Unsupported Versions] [answered]
Greetings.

When I run my old trusty Onion 16.04 command:

[root@search # **curl -k http://localhost:9200/_cat/indices**

I get the following response. 

**curl: (52) Empty reply from server**

so-status returns all normal. What might I be doing wrong?

Thanks, Judd

**Comment by juddcbrown** [comment, marked as answer]
Sorry, this is on the current Onion 2.0 search device.

**Reply by Git-Me-Some-Hub** [reply]
I think you want to use https
`curl -k https://localhost:9200/_cat/indices`

**Reply by juddcbrown** [reply]
Thank you!

**Reply by Git-Me-Some-Hub** [reply]
No problem. the -k is mandatory as it gets you past the prompt of https certificates.
